cli/main.cpp

```diff
@@ -95,6 +101,10 @@ int main(int argc, char* argv[])
     GetModuleFileNameA(nullptr, exename, sizeof(exename)/sizeof(exename[0])-1);
     argv[0] = exename;
 #endif
+#if defined(__APPLE__)
+	uint32_t size = sizeof(exename);
```

Comment:
Seems like there are tabs instead of spaces used here, which is why uncrustify fails.